packages/webpack/src/ember-webpack.ts

```diff
@@ -33,6 +34,7 @@ const debug = makeDebug('embroider:debug');
 // terser lazily so it's only loaded for production builds that use it. Don't
 // add any non-type-only imports here.
 import type { MinifyOptions } from 'terser';
+import { FingerprintOptions } from '@embroider/core/src/packager';
```

Comment:
Since this is just for types, maybe use:

```suggestion
import type { FingerprintOptions } from '@embroider/core/src/packager';
```